### DISCUSSION TITLE
Unicode

### DISCUSSION THREAD

**Discussion by midnightPush** [cssbattle/forum#256] [Q&A]
Hello fellows 😃 

something I don't understand working with unicode I want to use different color and font size, for example:
❤`<a>`❤`</a>`
`<p><a>`♞`</a>`♞

I select  `<a>` for color  `a{color: green} `and that's working
but for whole `<p>` = last 2 unicodes I would like different font-size
so if I make `p{font:50px''} ` only last one is changing and that uncode indside  `<p>><a>` tag is not
how do I do it ?
are there better selectors?

**Comment by Beowolve** [comment]
I see no problem
```html
❤<a>❤</a><p><a>♞</a>♞<style>body{background:#191919;color:fff;font:50q a}a{color:green}p{font:100px''}
```

![image](https://user-images.githubusercontent.com/32326640/165471423-63a82025-a645-4976-bd18-db4d97a3fb85.png)

**Reply by midnightPush** [reply]
yeah it does work with `body `or `*+*`, didn't work with just `*`

```❤<a>❤</a><p><a>♞</a>♞<style>*{background:#191919;color:fff;font:50q a}a{color:green}p{font:100px''}```

**Reply by gunnarbittersmann** [reply]
`p{font:100px''}` sets the font size for the knights, but for the first knight – the `a` inside the `p` – the rule `a{font:50q a}` applies and overwrites the font size.
With `*+*`, the selector doesn’t match and the `a` inherits the font size from its parent `p`.
